about.html

```diff
@@ -31,44 +31,54 @@
         <a href="index.html" onclick="closeNav()">Home</a>
       </div>
     </div>
-    <nav class="nav-container navbar-background" id="nav-icon">
-      <nav class="navbar navbar-light">
+    <nav class="nav-container navbar-background p-0" id="nav-icon">
+      <nav class="navbar navbar-light mobile-only">
         <button class="navbar-toggler col-2" onclick="openNav()">
           <span class="navbar-toggler-icon"></span>
         </button>
       </nav>
-      <nav class="menu-items">
-      <a href="#ai-logo" onclick="closeNav()">Angel Investor Logo</a>
-      <a href="#time-cards" onclick="closeNav()">Timeline</a>
-      <a href="#partner" onclick="closeNav()">Partner</a>
-      <a href="index.html" onclick="closeNav()">Home</a>
-      </nav>
+      <div class="w-100 d-nav align-items-center">
+        <img class=" col-3 mx-auto logo menu-items align-self-center pb-2" src="src/ailogo.png" alt="">
+        <nav class="menu-items col-8 align-content-center m-0 h-100 me-5">
+          <a href="#ai-logo" onclick="closeNav()">Angel Investor Logo</a>
+          <a href="#time-cards" onclick="closeNav()">Timeline</a>
+          <a href="#partner" onclick="closeNav()">Partner</a>
+          <a href="index.html" onclick="closeNav()">Home</a>
+          <button class="h-50 align-self-center secondary head-btn">AI
+            Campaign</button>
+        </nav>
+      </div>
+
     </nav>
   </header>
```

Comment:
- [ ]  Your navigation should be divided into top dark bar and bottom white bar as provided from Behance design 

Figma design 
![image](https://user-images.githubusercontent.com/43877689/181490290-24d618d9-e4e1-4b91-8814-d22e12850444.png)
Behance design 
![image](https://user-images.githubusercontent.com/43877689/181490875-053d7b8d-f3ba-48ca-8934-73f65d44cbb3.png)

Reply:
Addressed